### DISCUSSION TITLE
wiki/getting-started/change-themes/

### DISCUSSION THREAD

**Discussion by giscus** [PrismLauncher/prismlauncher.org#572] [Wiki]
# wiki/getting-started/change-themes/

An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.

https://prismlauncher.org/wiki/getting-started/change-themes/

**Comment by antoni0C0des** [comment]
IMO the themes install section is confusing. when u install a theme you must not have a folder inside a folder but it has to be a folder where all contents are inside of it (kinda like how you must see in MC's mod folder all jars inside & no jars inside other folders) I hope this is understandable lol

**Reply by AshtakaOOf** [reply]
You can always contribute to the documentation. :)

**Comment by Xarol9** [comment]
And how to change the location of the prism launcher to disk d

**Reply by blryface** [reply]
you do it in setup, but if you want to change where prism looks for instances and other folders it's in the launcher's settings

**Comment by loadfred** [comment]
DRACULA #1

**Comment by paworqtw** [comment]
Well it's unusable i downloaded "Tokyo-Night-theme" and do the extrat zip to  %appdata%\PrismLauncher\themes and then i restart the prism launcher(as all you can see it's not a background program so closing it means closing it and then i go into setting  user interface tab and click the icons nothing called "Tokyo night appeared

**Reply by paworqtw** [reply]
sorry it worked
